lib/src/model.dart

```diff
@@ -1277,8 +1283,49 @@ abstract class SourceCodeMixin {
     }).join('\n');
   }
 
+  String get crossdartHtmlTag {
+    if (config.addCrossdart && _crossdartUrl != null) {
+      return "<a class='crossdart' href='${_crossdartUrl}'>Link to Crossdart</a>";
+    } else {
+      return "";
+    }
+  }
+
+  int get _lineNumber {
+    var node = element.computeNode();
+    if (node is Declaration && (node as Declaration).element != null) {
+      return cache.lineNumber(
+          (node as Declaration).element.source.fullName, node.offset);
+    } else {
+      return null;
+    }
+  }
+
+  String get _crossdartUrl {
```

Comment:
Sorry I missed this before. Can you add some tests for this?

We don't need to test the generated HTML but it would be nice to test the output of calling `element.documentation` and `element.documentationAsHtml` with this option on and off.